index.html

```diff
@@ -22,13 +22,29 @@
     <main>
       <div class="image">
         <img src="./images/pass_photo.jpg" alt="Profile Photo" />
-        <div class="name-tag">
-          <h2>Qais Ziaey</h2>
-          <p>Webdeveloper</p>
-          <h4>Based in Bonn, Germany</h4>
-        </div>
       </div>
       <div class="portfolio_description">
+        <div class="address--info">
```

Comment:
If you want to use BEM notation here, then you would write "address__info" as class. Info is an element and not a modifier for highlighting